### DISCUSSION TITLE
Include postcode when looking up suburb

### DISCUSSION THREAD

**Discussion by SpitFire-666** [organicmaps/organicmaps#9351] [Ideas]
It would be handy showing the postcode when looking up a suburb. That is all :)

![image](https://github.com/user-attachments/assets/2b52b8ef-a7de-439c-9c51-8270742ca241)

![image](https://github.com/user-attachments/assets/d36ea645-f5b5-4c0e-837f-64daab43514e)

**Comment by patepelo** [comment]
It's very tricky since this data is not directly connected in OSM so I don't see it being incorporated any time soon.
